### ISSUE TITLE
v8 regression: fails evaluating when bundled

### ISSUE THREAD

**Issue by AviVahl** [postcss/postcss#1406]
Thanks for the new major! While I was playing with it, I've noticed v8.0.2 fails once bundled (using latest webpack@4).

Upon evaluation bundle throws:
```
TypeError: terminalHighlight.registerInput is not a function
```

This makes sense, as the `terminal-highlight` is being mapped to an empty object when postcss is bundled (`package.json->"browser"`).

Probably need to protect the call there and verify `terminalHighlight.registerInput` is defined first.

**Comment by ai** [comment]
Thanks. Fixed e8191be

**Comment by ai** [comment]
The fix was released in 8.0.3
